pyguitemp/threads.py

```diff
@@ -185,3 +187,58 @@ def abort_thread(self, ident: int) -> None:
     def post_event(self, event: ThreadResult):
         """Adds an event to the event loop of the main thread."""
         wx.PostEvent(self._window, event)
+
+
+def run_in_thread(
+    target: Callable,
+    on_abort: Optional[Callable] = None,
+    on_complete: Optional[Callable] = None,
+    on_error: Optional[Callable] = None,
+    daemon: Optional[bool] = None,
+) -> int:
+    """Is an alias for ThreadPool().run_thread(...)."""
+    return ThreadPool().run_thread(target, on_abort, on_complete, on_error, daemon)
+
+
+def run_daemon(
+    target: Callable,
+    on_abort: Optional[Callable] = None,
+    on_complete: Optional[Callable] = None,
+    on_error: Optional[Callable] = None,
+) -> int:
+    """Is an alias for ThreadPool().run_thread(..., daemon=True)."""
+    return ThreadPool().run_thread(target, on_abort, on_complete, on_error, daemon=True)
+
+
+def abort_thread(ident: int) -> None:
+    """Set thread with given identifier to abort.
+
+    Args:
+        ident: Thread identifier
+
+    Raises:
+        KeyError: If the identifier is not in the ThreadPool
+    """
+    if ident in ThreadPool()._workers:
+        ThreadPool().abort_thread(ident)
+    else:
```

Comment:
This is a very good point, but I would not put it here. Instead, checking if `ident` is actually a thread managed by the `ThreadPool` should be done within `ThreadPool().abort_thread` such that both functions - the original and the alias - behave exactly the same.